installer/templates/phx_test/controllers/page_controller_test.exs

```diff
@@ -2,7 +2,7 @@ defmodule <%= web_namespace %>.PageControllerTest do
   use <%= web_namespace %>.ConnCase
 
   test "GET /", %{conn: conn} do
-    conn = get conn, "/"
```

Comment:
I believe we want to allow `get/2` as a "DSL" call with no parens... Which I believe means it should be added to the `locals_without_parens` list in the `.formatter.exs` in the root of this repo.

I'm new here, double check. 😁

Reply:
@harlantwood `get/3` and `get/4` from [`Phoenix.Router`](https://github.com/phoenixframework/phoenix/blob/master/lib/phoenix/router.ex#L432)'s DSL are already added to `.formatter.exs`. This line here is in a test and uses `get/2` / `get/3` from [`Phoenix.ConnTest`](https://github.com/phoenixframework/phoenix/blob/master/lib/phoenix/test/conn_test.ex#L175), which is not a part of the Routing DSL.